arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "ServiceBusManagementClient",
     "description": "Azure Service Bus client",
-    "version": "2014-09-01"
+    "version": "2015-08-01"
```

Comment:
I will just rename the folder name. We do not need to keep the 2014-09-01 once you merge this change. Or I can do that right now as well.